src/core/tests/preprocess.cpp

```diff
@@ -2271,3 +2287,11 @@ TEST(pre_post_process, dump_error) {
     auto dump = stream.str();
     EXPECT_TRUE(dump.find("Error occurred:") != std::string::npos) << dump;
 }
+
+TEST(pre_post_process, preprocess_clamp) {
+    auto f = create_clamp(element::f16, Shape{1, 3, 2, 2}, 0.0f, 2.0f);
```

Comment:
The `create_clamp` helper looks like is not required the `create_simple_function` should be enough.

The test should take some model add the PPP should add clamp on some input, then assertion should detect that model is modified correctly.